src/joystick/hidapi/SDL_hidapi_gip.c

```diff
@@ -575,6 +575,11 @@ static bool GIP_SupportsVendorMessage(GIP_Device *device, Uint8 command, bool up
 static Uint8 GIP_SequenceNext(GIP_Device *device, Uint8 command, bool system)
 {
     Uint8 seq;
+
+    if (command == GIP_CMD_DIRECT_MOTOR) {
+        // The motor sequence is optional and always works with 0
+        return 0;
+    }
```

Comment:
This belongs in the else block below, not up here. You should probably change the comment to be "sequence number" too.